website/content/en/highlights/2021-08-24-vector-aggregator.md

```diff
@@ -0,0 +1,48 @@
+---
+date: "2021-08-24"
+title: "Moving the aggregator chart to public beta"
+description: "The Vector aggregator Helm chart is now publicly available"
+authors: ["spencergilbert"]
+pr_numbers: [8801]
+release: "0.16.0"
+hide_on_release_notes: false
+badges:
+  type: "announcement"
+  platforms: ["helm"]
+---
+
+We are happy to announce that our [`vector-aggregator`][chart] chart is now publicly available.
+While we continue to iterate on and make improvements to the chart, we feel it is ready to get
+wider feedback from the community at large.
+
+We have also created a dedicated Discord channel, [#aggregator][discord], for support, questions,
+and feedback.
+
+## What is an aggregator
+
+When Vector is run in an [aggregator role][aggregator], it is used to transform and ship data
```

Comment:
```suggestion
When Vector is deployed as an [aggregator][aggregator], it is used to transform and ship data
```

Reply:
I'll wait for @binarylogic's feedback and batch this with it (if needed)